shared/src/common/enums/user/user-gender.enum.ts

```diff
@@ -1,7 +1,7 @@
 enum UserGender {
-  MALE = 'male',
-  FEMALE = 'female',
-  OTHER = 'other',
+  MALE = 'Male',
+  FEMALE = 'Female',
+  OTHER = 'Other',
```

Comment:
```suggestion
  MALE = 'male',
  FEMALE = 'female',
  OTHER = 'other',
```
should be handled by styles